build.gradle

```diff
@@ -179,11 +179,14 @@ configure(opentelemetryProjects) {
     }
 
     dependencies {
-
-        implementation platform(boms.grpc),
-                platform(boms.guava),
-                platform(boms.protobuf),
-                platform(boms.zipkin_reporter)
+        configurations.all {
+            if (it.name.endsWith('Classpath')) {
+                add(it.name, platform(boms.grpc))
+                add(it.name, platform(boms.guava))
+                add(it.name, platform(boms.protobuf))
+                add(it.name, platform(boms.zipkin_reporter))
```

Comment:
maybe we should use here forcedPlatform instead of platform what's ur opinion @anuraaga

Reply:
Yeah I usually use enforcedPlatform for this sort of management - switched to it